docs/providers/documentation/splunk-provider.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: "Splunk"
+sidebarTitle: "Splunk Provider"
+description: "Splunk provider allows you to query Splunk metrics and logs for monitoring and analytics."
```

Comment:
btw, this is incorrect. We currently just support getting "saved search" alerts

Reply:
:) Will fix this.